### PR TITLE
[do not merge] redis dev environment

### PR DESCRIPTION
**Pull request by tarekziade**
This provides a full redis environment for development purpose.

### CODE REVIEW

**Comment by tarekziade** [comment]
Turns out there are deeper problems with this.

We need to:
1- allow running and launching tests in cliquet w/o postgres.
2- allow running and launching tests in kinto w/o postgres.

change the Tox file so it tries both environement. Then offer the two configs like in my initial PR

**Comment by ametaireau** [comment]
To run the tests, you should be able to run postgres. I think this needs to stay as a requirement since it's the default database we're using.

File: config/kinto-dev.ini [hunk 1]
```diff
@@ -0,0 +1,53 @@
+[app:main]
```
Comment (ametaireau) [inline config/kinto-dev.ini]: We should name this kinto-redis

**Comment by ametaireau** [comment]
IMO the default env for devs is to use postgres, since it's the target database.

Installing it with docker is a matter of minutes. In case we really want to go down this path, then agree, we should change this, but I don't think that's the path we want to take.

**Comment by leplatrem** [comment]
https://github.com/mozilla-services/cliquet/issues/124

**Comment by Natim** [comment]
For kinto the default backends are postgresql.
The documentation explains how this can be changed.

**Pull request closed without merging**